docs/content/en/docs/release-notes/_index.md

```diff
@@ -0,0 +1,123 @@
+
+---
+title: "K8ssandra release notes"
+linkTitle: "Release notes"
+weight: 1
+description: Release notes for the open-source K8ssandra community project.
+---
+
+K8ssandra provides a production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. 
+
+Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API. 
+
+{{% alert title="Note" color="success" %}}
+The **K8ssandra 1.1.0** release implements a number of changes, enhancements, and bug fixes. This Release Notes topic lists a subset of the key updates. For the complete list, see the [CHANGELOG](https://github.com/k8ssandra/k8ssandra/blob/main/CHANGELOG.md).  
+{{% /alert %}}
+
+**Release date:** 09-April-2021.
+
+## Prerequisites
+
+* A Kubernetes v1.16 or later environment - local or via a supported cloud provider
+* [Helm](https://helm.sh/) v3
+
+## Supported Kubernetes environments
+
+* Open-source [kubernetes.io](https://kubernetes.io)
+* [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE)
+* [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/) (AKS)
+* [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS)
+* [MiniKube](https://minikube.sigs.k8s.io/docs/)
+* [Kind](https://kind.sigs.k8s.io/)
+* [K3D](https://k3d.io/)
+
+## K8ssandra deployed components
+
+The K8ssandra helm chart deploys the following components. Some are optional, and depending on the configuration, may not be deployed:
+
+* Apache Cassandra - the deployed version depends on the configured `serverImage` setting:
+  * 3.11.7
+  * 3.11.8
+  * 3.11.9
+  * 3.11.10 (default)
+* DataStax Kubernetes Operator for Apache Cassandra (cass-operator) 1.6.0
+* Management API for Apache Cassandra (MAAC) 0.1.24
+* Stargate 1.0.11
+* Metric Collector for Apache Cassandra (MCAC) 0.2.0
+* kube-prometheus-stack 12.11.3 [chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+* Medusa for Apache Cassandra 0.10.0
+* Reaper for Apache Cassandra 2.2.2
+
+## Upgrade notice
+
+{{% alert title="Important!" color="warning" %}}
+Upgrading from K8ssandra 1.0.0 to 1.1.0 causes a StatefulSet update (due to [#533](https://github.com/k8ssandra/k8ssandra/issues/533) and [#613](https://github.com/k8ssandra/k8ssandra/issues/613)). A StatefulSet update has the effect of a rolling restart. Because of [#411](https://github.com/k8ssandra/k8ssandra/issues/411) this could require you to perform a manual restart of all Stargate nodes after the Cassandra cluster is back online. This behavior also impacts in-place restore operations of Medusa backups [#611](https://github.com/k8ssandra/k8ssandra/issues/611). 
+
+To manually restart Stargate nodes, on each node:
```

Comment:
These commands only need to be run once. You can change line 56 to just be `To manually restart Stargate nodes:`

Reply:
Done